Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -113,7 +116,7 @@ def validate_traffic_analysis_arguments(port: Optional[int], policy_decisions: l
         raise InvalidValueError("protocol", protocol, VALID_PROTOCOLS)
 
 
-def validate_virtual_service_arguments(port: int | None, protocol: str) -> None:
+def validate_virtual_service_arguments(port: int, protocol: str) -> None:
```

Comment:
Prior to 3.10, the way to represent `type | None` is generally `Optional[type]`

Reply:
Made the changes